examples/experimental/onnx/classification/onnx_ptq_classification.py

```diff
@@ -38,15 +39,17 @@ def run(onnx_model_path: str, output_model_path: str,
     original_model = onnx.load(onnx_model_path)
     nncf_logger.info("The model is loaded from {}".format(onnx_model_path))
 
+    assert input_shape or input_keys, "Either input_shape or input_keys must be set."
+
     if input_shape is None:
         nncf_logger.info(
             "input_shape is None. Infer input_shape from the model.")
-        input_shape = infer_input_shape(original_model)
 
-    input_keys = [node.name for node in original_model.graph.input]
-    if len(input_keys) != 1:
-        raise RuntimeError(
-            f"The number of inputs should be 1(!={len(input_keys)}).")
+    elif input_keys is None:
+        nncf_logger.info(
```

Comment:
Could you place this logging inside the function infer_input_shape()?

Reply:
Done